### ISSUE TITLE
Asian-American Quality of Life / Community Engagement "Tea in a Box"

### ISSUE THREAD

**Issue by CPIOBenAdams** [open-austin/project-ideas#32]
**Concise description**: 
The City of Austin is working on a community engagement initiative that seeks to engage with the City's Asian-American population (which is varied and diverse). As part of this project we plan to institute a project called "Tea in a Box," modeled after the popular "Budget in a Box" and "Meeting in a Box" initiatives that have been sweeping the country in the private and non-profit sectors. 

Essentially, we will be designing a box (dimensions 12x9x4") which will include a series of items, including a tea cup, tea bags, markers, pens, a survey, DVD-video (outlining the idea of the project), comment cards, colored sticker dots, facilitator instructions, and a game board. The idea will be for the City of Austin to deploy facilitators into the Asian-American community in Austin. These facilitators will facilitate the engagement activity. 

We have identified 5 pillars that we will emphasize when engaging with community members: Economic Development, Housing and Community Development, Health and Human Services, Arts and Culture, and Civic Engagement. 

We plan to incorporate an artistic element into this project. That is, we plan on including a game board, divided into a grid-like pattern. Each box in the grid will highlight one of the 5 pillars (listed above) that we have prioritized. While the exact details of the game have not yet been fine-tuned, we do anticipate asking the Asian-American participants to place a colored sticker (Green=High Priority, Yellow=Medium Priority, and Red=Low Priority) on each of the squares in the box. This will allow us to get a better idea of which of the pillars the City of Austin should be paying the most attention to. Further, as mentioned above, we intend to include some kind of artistic element into this project. As an example, we might have some magazines that the Asian-American participants could cut pictures out of, creating a collage of sorts, allowing them to identify areas of concern in their lives and in the City's administration of services to them. 

We are currently in the process of nailing down some of the finer details of the artistic element of this project. But we are certain that we will need a mobile app that would allow some participants to engage in this activity from the comfort of their homes and without the assistance of a City of Austin facilitator. 

In short, we would like to have a mobile app that would incorporate the themes, game/artistic concept, survey, and informational materials into its user interface. And that's where we would love to have your technical support!
**Link (more details/brain dump/alpha)**

**Who will use/benefit from this project?**
Austin's Asian-American population. 

**Project Needs (dev/design/resources)**
Development of an application that will incorporate the themes, game/artistic concept, survey, and informational materials into its user interface.

**Status (in progress, pie-in-the-sky)**
This project is underway. The City of Austin will definitely be implementing this project.

**Comment by luqmaan** [comment]
Dupe of #31.
